app/views/admin/docs/edit.html.erb

```diff
@@ -1,9 +1,9 @@
 <h2 class="form-subhead"><%= t(:edit, default:'Edit') %>: <%= @doc.title %></h2>
 <%= locale_select if AppSettings['i18n.available_locales'].count > 1 %>
-<%= bootstrap_form_for :doc, url: admin_category_doc_path(category_id: @category.id, id: @doc.id), :method => 'patch' do |f| %>
+<%= simple_form_for :doc, url: admin_category_doc_path(category_id: @category.id, id: @doc.id), :method => 'patch' do |f| %>
 
   <% Globalize.with_locale(params['lang']) do %>
-  <%= f.alert_message "Please fix the errors below." %>
+  <%= render 'shared/error_messages', object: f.object %>
   <%= render :partial => 'form', :object => @doc, :locals => {:f => f} %>
   <% end %>
 <% end %>
```

Comment:
I also noticed that for some reason, all form fields are picking up an asterisk indicating they are required even though they are not, and the booleans are showing up as text_fields:

![screen shot 2016-06-03 at 11 58 53 am](https://cloud.githubusercontent.com/assets/14369/15788118/105a93ae-2983-11e6-8c5a-60a82b7c0724.png)

This seems to only occur on the edit view, not the new view.